### ISSUE TITLE
Incompatibility with the App Immuni

### ISSUE THREAD

**Issue by OlafRadicke** [corona-warn-app/cwa-app-android#700]
## Avoid duplicates
* [ ] Bug is not mentioned in the [FAQ](https://www.coronawarn.app/en/faq/)
* [ ] Bug is specific for Android only, for general issues / questions that apply to iOS and Android please raise them in the [documentation repository](https://github.com/corona-warn-app/cwa-documentation)
* [ X] Bug is not already reported in another issue

## Describe the bug

I run you App for some days and all was fine. Than I installed the Add "Immuni" (https://play.google.com/store/apps/details?id=it.ministerodellasalute.immuni). The first look was fine too, but one day later I open the  corona-warn-app and see that tracking  has stopped. I'v see the message: Corona-Warn-App does not track your location and doesn't have the permission to do this. So I switch this on again. But now, I get the same warning from the other App  "Immuni". With other word: it can be run only one of this App. 

## Steps to reproduce the issue

- Install and start corona-warn-app
- Install "Immuni" (https://play.google.com/store/apps/details?id=it.ministerodellasalute.immuni). 
- wait 10 minutes
- Reopen corona-warn-app and you will see, that is stopped.
- Reactivate corona-warn-app
- Switch back to "Immuni" and see the warning and that the App is stopped now

## Technical details

- Mobile device: Nokia 7 plus
- Android version: 10

## Additional context

- It's not rooted device
- All Apps are downloadetd from Playstor

**Comment by mh-** [comment]
This is a limitation introduced by Apple and Google - only 1 app can connect to their Exposure Notifications API on each device. I believe they do this to keep complexity for their interfaces low.
Also they have limited the rate how often an app can access the API, for user privacy reasons; and two apps sharing this limit wouldn't work, while doubling the limit for two apps could circumvent the restrictions.

**Comment by OlafRadicke** [comment]
> This is a limitation introduced by Apple and Google - only 1 app can connect to their Exposure Notifications API on each device. I believe they do this to keep complexity for their interfaces low.
> Also they have limited the rate how often an app can access the API, for user privacy reasons; and two apps sharing this limit wouldn't work, while doubling the limit for two apps could circumvent the restrictions.
> 
> You could however switch between apps (e.g. when you go on vacation in Italy), and they will access the same collected data on the device, just not both apps in parallel. So be careful, Immuni will only tell you about warnings by Italian users, and Corona-Warn-App will only tell you about warnings by German users.

Okay, but then you have a blind spot of three weeks after your Italy vacation. So you have to make a decision. Do you protect your Italian hosts or your German friends and family Who are you going to sacrifice?

**Comment by mh-** [comment]
I think that if you were tested positive, you could use both apps to upload your TEKs, one after the other. For Corona-Warn-App you need to obtain a teleTAN, for Immuni, I don't know how they check that you have been tested positive.

**Comment by OlafRadicke** [comment]
> I think that if you were tested positive, you could use both apps to upload your TEKs, one after the other. For Corona-Warn-App you need to obtain a teleTAN, for Immuni, I don't know how they check that you have been tested positive.

Okay, if you turn off the Immuni App in Germany, do you still get information about critical contacts in Italy? I think not. If access to the device is exclusive. Then it might make sense to build an API gateway for the notifications to and from Italy (and all other countries).

**Comment by OlafRadicke** [comment]
Addendum: With Immuni you must be in Italy to upload your code to indicate a positive test. But you will certainly not want to go back to Italy if you are dead sick. Just to upload your code.  

I see no alternative to API gateways.

**Comment by DevFlamur** [comment]
What's the deal of this issue? why would you want to use both at the same time if you live in Germany?
I hope this discussion is not leading to have several apps for each country.. where as the apps can't live together in one single device, because in Germany there aren't only Germans but Italians as well 🤔.
But in the other point you are right, there should be an list of apps which Corona App can not live together with, like "Immuni" because of Exposure notification API.. Simple users doesn't understand this, they need an info at least 😊

**Comment by OlafRadicke** [comment]
> What's the deal of this issue? why would you want to use both at the same time if you live in Germany?
> I hope this discussion is not leading to have several apps for each country.. where as the apps can't live together in one single device, because in Germany there aren't only Germans but Italians as well thinking.
> But in the other point you are right, there should be an list of apps which Corona App can not live together with, like "Immuni" because of Exposure notification API.. Simple users doesn't understand this, they need an info at least blush

Oh, remember Ischgl! Hundreds of Germans spent their holidays there and brought the virus home. Next winter - same game. What help does the app offer if the problem is not solved? Right, none.

**Comment by MikeJayDee** [comment]
At the moment, you can install both and then activate them alternatingly manually every day or so to receive warnings from both apps.

However, it has been communicated that work is underway so that the backends of the apps using GAEN will exchange keys somehow.

**Comment by DevFlamur** [comment]
@OlafRadicke Agree the App does not solve the 'corona' or any sickness problem because it's an health problem, but it at least help if used properly to 'slow down' the infection in order that the hospitals will not be overloaded. But with the app the real problem is not Solved!

**Comment by MikeJayDee** [comment]
Flamur <notifications@github.com> schrieb am So., 21. Juni 2020, 22:37:

**Comment by tkowark** [comment]
Thanks for bringing up these points. That's also more a general issue not only focused on the Android App. Hence, I'll move the issue to the documentation repo.
